src/imgui/ImGuiMachine.cc

```diff
@@ -46,9 +46,63 @@ ImGuiMachine::ImGuiMachine(ImGuiManager& manager_)
 	, setupFileList("setup", Reactor::SETUP_EXTENSION, Reactor::SETUP_DIR)
 	, confirmDialog("Confirm##setup")
 {
-	setupFileList.selectAction = [&](const FileListWidget::Entry& entry) {
+
+	setupFileList.drawAction = [&] {
+		im::Table("table", 2, ImGuiTableFlags_BordersInnerV, [&]{
+			if (ImGui::TableNextColumn()) {
+				setupFileList.drawTable();
+			}
+			if (ImGui::TableNextColumn()) {
+				if (previewSetup.motherBoard) {
+					showSetupOverview(*previewSetup.motherBoard);
+					ImGui::Separator();
+					if (ImGui::Button("Run!")) {
+						manager.executeDelayed(makeTclList("setup", previewSetup.displayName));
+					}
```

Comment:
Detail: does this "Run" button belong below the preview section, or below the file list? My first idea would be the file list, but I haven't tried your code yet (I haven't actually seen how it looks).

Reply:
Please check how it looks and feels. To me it makes sense (as an alternative to double-click) to click a Run! button on the thing you see on your screen to directly run it. Especially as the double-click may not be intuitive.

Reply:
I moved it to keep it in a fixed place. Good argument.